### PR TITLE
RF : new trainning with DR8

### PR DESCRIPTION
**Pull request by echaussidon**
Hi all,

I trained two new Random Forests with DR8 photometry, they are added in the same repository than the others. 

The RF selection  is the same, when  'releaseReduced >= 8000' we use now DR8 RF.

Edmond

### CODE REVIEW

**Comment by echaussidon** [comment]
I think I need two reviews to be merged : @Cyeche @geordie666 

Thanks

**Comment by Cyeche** [comment]
That's fine with me.

I think that at the same time we should update the definition of the bit mask:
https://github.com/desihub/desitarget/blob/master/py/desitarget/cuts.py#L1310

We should replace 
[1, 10, 12, 13]:
by
[1, 5, 6, 7, 10, 12, 13]:

Thanks

File: py/desitarget/myRF.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def predict_proba(self):
 
     def loadForest(self, forestFileName):
         # loads forest
-        t = np.load(forestFileName, encoding='bytes')
+        t = np.load(forestFileName, encoding='bytes', allow_pickle=True)
```
Comment (geordie666) [inline py/desitarget/myRF.py:93]: Given that `allow_pickle` has not been needed to load the RF files in the past, is it really needed now? In general, `.npz` files, although a little module/language specific, should be supported by `numpy.load()` for the foreseeable future. But, the pickled versions are less secure, less likely to fail with meaningful errors on erroneous data, and have an extra dependency (on the pickle module). I'd prefer it if the DR8 RF files didn't require `allow_pickle`.

Comment (echaussidon) [reply py/desitarget/myRF.py]: It seems that `numpy.save_compressed()` use pickle automatically with file heavier than 12 MB. This why, I needed that. According to the documentation, this just allows file decompression using pickle and does not change the rest.

Whatever, with new RF files which are lighter, it's not require. I **remove** it.

File: py/desitarget/cuts.py [hunk 2]
```diff
@@ -1373,6 +1375,28 @@ def isQSO_randomforest(gflux=None, rflux=None, zflux=None,
...
 
+        tmpReleaseOK = releaseReduced >= 8000
+        if np.any(tmpReleaseOK):
+            # rf initialization - colors data duplicated within "myRF"
+            rf = myRF(colorsReduced[tmpReleaseOK], pathToRF,
+                      numberOfTrees=500, version=2)
+            rf_HighZ = myRF(colorsReduced[tmpReleaseOK], pathToRF,
+                            numberOfTrees=500, version=2)
+            # rf loading
+            rf.loadForest(rf_DR8_fileName)
```
[the remark this answers is not in the document]
Comment (echaussidon) [reply py/desitarget/cuts.py:1386]: New files respect this condition.

**Comment by echaussidon** [comment]
I add mask as @Cyeche suggests

File: py/desitarget/cuts.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (geordie666) [reply py/desitarget/cuts.py:1387]: These look good, thanks.

**Comment by geordie666** [comment]
Thanks @echaussidon. This all looks good, now. I'm running some simple checks that everything runs to completion and then I'll merge this PR.

For the record: I'm then going to make a new tag of `desitarget` and run a full suite of DR8 targeting files. This is to ensure that @forero has more realistic target files for his work on simulating fiber assign for the Main Survey.

Also, technically it's always good practice to update the [changes document:](https://github.com/desihub/desitarget/blob/master/doc/changes.rst) when submitting a PR. But, in this case, I'll update the changes myself when I make the new tag.

**Pull request merged**